lib/util.js

```diff
@@ -496,6 +496,14 @@ function getConstructorName(obj) {
   return '';
 }
 
+// Checks and returns [<type>: null prototype]
+// if object has null prototype
+function checkNullPrototype(constructor, type) {
+  if (constructor === '')
+    return `[${type}: null prototype]`;
+  return constructor;
```

Comment:
Please inline this check into the `getPrefix` check. That way the code should be simpler and pretty straight forward.

Reply:
Yeah, thought about it. Ok, will work on that and update the PR.